src/themes/teams/components/Icon/svg/ProcessedIcons/icons-settings.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react'
+import { TeamsSvgIconSpec } from '../types'
+
+export default {
+  icon: ({ classes }) => (
+    <svg role="presentation" focusable="false" viewBox="8 8 16 16" className={classes.svg}>
+      <path
+        className={classes.outlinePart}
+        d="M21.93 15.719v.564l2.023 1.258-1.242 2.992-2.32-.533q-.191.2-.391.391l.531 2.32-2.992 1.242-1.257-2.023h-.564l-1.258 2.023-2.992-1.242.532-2.32q-.2-.191-.39-.391l-2.32.531-1.243-2.992 2.023-1.258v-.141-.141-.141-.141l-2.023-1.256 1.243-2.992 2.32.531c.125-.135.255-.266.391-.391l-.531-2.32 2.992-1.242 1.258 2.023h.564l1.257-2.023 2.992 1.242-.533 2.32c.136.125.266.256.391.391l2.32-.531 1.242 2.992zm-.914 1.016l.031-.371c.01-.123.016-.246.016-.371s0-.242-.016-.367-.021-.247-.031-.367l1.844-1.148-.672-1.625-2.117.492a6.771 6.771 0 0 0-.5-.547q-.253-.249-.551-.5l.492-2.117-1.625-.672-1.156 1.844-.367-.031c-.125-.01-.248-.016-.367-.016s-.249.006-.371.016l-.371.031-1.145-1.845-1.625.672.492 2.117a6.889 6.889 0 0 0-.547.5q-.25.255-.5.551l-2.117-.492-.672 1.62 1.844 1.156-.031.371a4.544 4.544 0 0 0 0 .738c.016.125.021.248.031.367l-1.844 1.15.672 1.625 2.117-.492a6.54 6.54 0 0 0 .5.547q.253.25.551.5l-.492 2.117 1.625.672 1.156-1.844.367.031c.125.011.247.016.367.016s.249 0 .371-.016l.371-.031 1.148 1.844 1.625-.672-.496-2.118a6.456 6.456 0 0 0 .547-.5q.25-.253.5-.551l2.117.492.672-1.625zM16 13.063a2.865 2.865 0 0 1 1.144.23 2.941 2.941 0 0 1 1.563 1.563 2.957 2.957 0 0 1 0 2.289 2.941 2.941 0 0 1-1.562 1.563 2.956 2.956 0 0 1-2.289 0 2.941 2.941 0 0 1-1.562-1.562 2.957 2.957 0 0 1 0-2.289 2.941 2.941 0 0 1 1.563-1.562A2.867 2.867 0 0 1 16 13.063zm0 5a2.017 2.017 0 0 0 .8-.16 2.06 2.06 0 0 0 1.1-1.1 2.1 2.1 0 0 0 0-1.609 2.06 2.06 0 0 0-1.1-1.1 2.1 2.1 0 0 0-1.609 0 2.059 2.059 0 0 0-1.1 1.1 2.1 2.1 0 0 0 0 1.609 2.059 2.059 0 0 0 1.1 1.1 2.015 2.015 0 0 0 .809.16z"
+      />
+      <path
+        className={classes.filledPart}
+        fill-rule="evenodd"
```

Comment:
note, this should be defined as `fillRule`, no dash is allowed - similar to `border-color` vs `borderColor`. Currently this results in runtime error.